src/Pages/EmailList/EmailTemplate.js

```diff
@@ -0,0 +1,294 @@
+        allEmail.push(item.email);
+        memberEmail.push(item.email);
+      }
+    }, this.setState({ officerEmail, memberEmail, allEmail }));
+  };
+
+  getEvents = async () => {
+    const apiResponse = await getAllEvents();
+    if (!apiResponse.error) {
+      this.storeContent(apiResponse);
+    } else {
+      let errorMessage = 'Unable to retrieve events.';
+      this.setState({ events: [errorMessage] });
+    }
+  };
+
+  // customize data with html, append header/footer, and store that to 'content'
+  storeContent = async (events) => {
+    const mailHeader =
+      '<p><span style="font-size: 10pt; font-family: Arial;">' +
+      '<strong>Hello SCE G4NG,</strong></span></p>' +
+      '\n<p><span style="font-size: 10pt; font-family: Arial;">' +
+      '<strong>This is our newsletter, please read ty.</strong></span></p>\n';
+
+    const mailFooter =
+      '<p><strong><span style="color: #236fa1; font-size: 10pt;">Software ' +
+      'and Computer Engineering</span><span style="color: #236fa1; ' +
+      'font-size: 10pt;"> Society<br /></span></strong>' +
+      '<span style="font-size: 10pt;">Website: ' +
+      '<a href="http://sce.engr.sjsu.edu">http://sce.engr.sjsu.edu</a><br />' +
+      'Contact: <a href="mailto:sce.sjsu@gmail.com">sce.sjsu@gmail.com</a>' +
+      '<br />Facebook: <a href="https://www.facebook.com/sjsusce/">@scesjsu' +
+      '</a><br />Instagram: <a href="https://www.instagram.com/sjsusce/">' +
+      '@scesjsu</a><br />CmpE Slack: <a href="https://cmpesjsu.slack.com/?' +
+      'redir=%2Fmessages%2FCH6TTGXLG#/">#sce</a></span></p>';
+    let data = undefined;
+
+    try {
+      let x = 0;
+      for (x = 0; x < events.responseData.length; x++) {
+        let responseData = events.responseData[x];
+        let eventDateFormatted = this.returnProperFormattedDate(
+          responseData.eventDate.toString().substring(0, 10)
+        );
+
+        const dataSingle =
+          '<p><br /><span style="font-size: 14pt; color: #0055a2;"><strong>' +
+          responseData.title +
+          '</strong></span><br /><span style="font-size: 10pt;">' +
+          responseData.description +
+          '</span><br /><br />' +
+          '<span style="font-size: 10pt;"><strong>When:</strong>&nbsp;' +
+          eventDateFormatted +
+          '&nbsp;at&nbsp;' +
+          responseData.startTime.replace(/ /g, '') +
+          '-' +
+          responseData.endTime.replace(/ /g, '') +
+          '</span><br /><span style="font-size: 10pt;"><strong>Where:' +
+          '</strong>&nbsp;' +
+          responseData.eventLocation +
+          '</span></p>\n<p><img style="max-height: 192px;' +
+          ' max-width: 192px;" src="' +
+          responseData.imageURL +
+          '" /></p>\n';
+
+        if (x === 0) {
+          data = dataSingle;
+        } else {
+          data = data + dataSingle;
+        }
+      }
+      data =
+        mailHeader +
+        data +
+        '<p>𝔁𝓾𝓮🥶𝓱𝓾𝓪🧚𝓹𝓲𝓪𝓸😻𝓹𝓲𝓪𝓸🗿𝓫𝓮𝓲👺𝓯𝓮𝓷𝓰🤩' +
```

Comment:
lol do we still need this

Reply:
Made it more professional looking for the meeting sunday 8)